mousestyles/dynamics/tests/test_dynamics.py

```diff
@@ -37,3 +41,59 @@ def test_creat_time_matrix():
     # Checking functions output the correct time matrix
     matrix = create_time_matrix(combined_gap=4, time_gap=1, days_index=0)
     assert matrix.iloc[0, 2181] == 1.0
+
+
+def test_get_prob_matrix_list_input():
+    # checking functions raise the correct errors for wrong input

```

Comment:
The wrong input here are just examples of inputs that are not desired by the function. This test function will check if each wrong input will raise the correct error message designed for that kind of wrong input.